packages/vaadin-lumo-styles/global.css

```diff
@@ -4,5 +4,20 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 @import './src/global/badge.css';
-@import './src/global/color.css';
+@import './src/global/dark.css';
 @import './src/global/typography.css';
+
+:where(:root, :host) {
+  color: var(--lumo-body-text-color);
+  background-color: var(--lumo-base-color);
+  color-scheme: light;
+}
+
+:where(body, :host) {
```

Comment:
What was the reason for moving this outside of `typography.css`?

Reply:
Oh, that's not required anymore. I moved it back to `typography.css`